### ISSUE TITLE
Would it be possible to implement the DR-SC in Seurat?

### ISSUE THREAD

**Issue by ElyasMo** [feiyoung/DR-SC.Analysis#2]
Would it be possible to implement the DR-SC in Seurat?

**Comment by feiyoung** [comment]
Yes, DR.SC package has the interface for the Seurat object. You can refer the vignettes for usage at https://feiyoung.github.io/DR.SC/articles.
